conftest.py

```diff
@@ -70,14 +70,17 @@ def pytest_exception_interact(node, call, report):
             test_name = node.name
             test_name = sanitize_filename(test_name)
             logging.error(f"Handling exception for test: {test_name}")
-            logging.error(
-                f"NODE LOGS HERE {node.funcargs}\n THE FAILED TEST: {test_name}"
-            )
-            driver = node.funcargs.get("driver")
-            opt_ci = node.funcargs.get("opt_ci")
-            if driver:
-                log_content(opt_ci, driver, test_name)
-                screenshot_content(driver, opt_ci, test_name)
+            if hasattr(node, "funcargs"):
```

Comment:
im acc sad that this isnt always defined during runtime ;-; pytest whyyy

Reply:
If the test doesn't get collected properly, the object passed to the hook is `Collector`, which does not have `funcargs`. If the test was collected, it's the normal test `Node` or `Item` or whatever it's called.